src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -65,7 +63,7 @@ def target_package
       end
 
       def find_or_create_branched_package
-        return target_package if updated_pull_request? && target_package.present?
+        return target_package if WorkflowEventAndActionValidator.new(scm_extractor_payload: scm_extractor_payload).updated_pull_request? && target_package.present?
```

Comment:
this `WorkflowEventAndActionValidator.new(scm_extractor_payload: scm_extractor_payload)` could be moved to a method no?

Reply:
Yeah, I'll move it